dist/init/linux-upstart/caddy.conf

```diff
@@ -14,12 +14,12 @@ respawn limit 10 5
 reload signal SIGUSR1
 
 # Let's Encrypt certificates will be written to this directory.
-env CADDYPATH=/etc/caddy
+env CADDYPATH=/etc/ssl/caddy
```

Comment:
I'm just an outsider (I don't use these init scripts), but `/etc/ssl/caddy` is a weird path. Why not just `/etc/caddy`? SSL is an old technology anyways.